src/pages/Movement/Movement.tsx

```diff
@@ -51,6 +52,10 @@ const Movement: React.FC = () => {
     return <Styles.Chip label={displayAmount} chipcolor={amountColor} />;
   };
 
+  const generateClickableId = (route: string, id: string | number, value: string | number) => {
```

Comment:
Let's create from this common component inside `src/components` looks like to it will be re-used in several places

Reply:
sure, done